documentation/src/main/resources/pages/ditto/baic-auth-checkpermissions.md

```diff
@@ -2,10 +2,10 @@
 title: Checking Permissions for Resources  
 keywords: permissions, authorization, resources, policy, checkPermissions  
 tags: [model]  
-permalink: check-permissions.html  
+permalink: basic-auth-checkpermissions.html  
```

Comment:
the filename is still wrong:
`baic-auth-checkpermissions.md` (missing the "s" in basic)